src/components/ResultView.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default class ResultView extends React.Component {
+  render() {
+    return (
+            <div className="col-md-12 resultview">
+              <p className='result'>{this.props.result}</p>
+            </div>
```

Comment:
It looks fantastic, but to better match the design I'd highly recommend increasing the height of your result view and making it as much as your button's height.

![image](https://user-images.githubusercontent.com/34779928/204633511-788f4e90-7432-4aa3-8ab8-76ea72f56818.png)


[Optional] You'll need to write and read from this component, so don't you think it must be an `input` tag with the type of `text or number`?